src/utils/filters.js

```diff
@@ -44,7 +44,16 @@ export function processFilters(layerData, filterOptions) {
   } else if (layerData.aggregate.filter) {
     for (f = 0; f < layerData.aggregate.filter.length; f += 1) {
       if (acceptedFilterValues[f] !== 'all' && acceptedFilterValues[f] !== 'quant') {
+<<<<<<< Updated upstream
         data = data.filter(filterProcessor);
+=======
+        if (acceptedFilterValues.filter(a => Array.isArray(a) && a.length).length > 1 && isOr) {
+          Data.filter(filterProcessor).map(d => combinedData.push(d));
+          data = [...new Set([...combinedData])]; // achieve a distinct union
+        } else {
+          data = data.filter(filterProcessor);
+        }
+>>>>>>> Stashed changes
```

Comment:
Looks like `Stashed changes` is newer code, right @KipSigei?

Reply:
I will resolve them